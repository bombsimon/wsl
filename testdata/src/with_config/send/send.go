package testpkg

func fn1(ch chan int) {
	a := 1
	ch <- a

	b := 1
	ch <- someFn(b)

	c := 1
	ch <- 1 // want `missing whitespace decreases readability \(send\)`

	// TODO: Technically this is used first in block but there's no easy way to
	// figure out the first statement in the block so for now this is not valid.
	d := 2
	ch <- func() int { // want `missing whitespace decreases readability \(send\)`
		return d
	}()
}
