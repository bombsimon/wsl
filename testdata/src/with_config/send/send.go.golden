package testpkg

func fn1(ch chan int) {
	a := 1
	ch <- a

	b := 1
	ch <- someFn(b)

	c := 1

	ch <- 1 // want `missing whitespace above this line \(no shared variables above send\)`

	d := 2

	ch <- func() int { // want `missing whitespace above this line \(no shared variables above send\)`
		return c
	}()

	e := 2
	ch <- func() int {
		return e
	}()
}
